getting_started/workflow/export/exporting_for_web.rst

```diff
@@ -4,12 +4,40 @@ Exporting for the Web
 =====================
 
 HTML5 export allows publishing games made in Godot Engine to the browser.
-This requires support for the recent technologies `WebAssembly
-<https://webassembly.org/>`__ and `WebGL 2.0 <https://www.khronos.org/webgl/>`__
-in the user's browser. **Firefox** and **Chromium** (Chrome, Opera) are
-the most popular supported browsers, **Safari** and **Edge** do not work yet.
-On **iOS**, all browsers must be based on WebKit (i.e. Safari), so they will also
-not work.
+This requires support for `WebAssembly
+<https://webassembly.org/>`__ and `WebGL <https://www.khronos.org/webgl/>`__
+in the user's browser.
+
+.. Important::
+ Use the browser-integrated developer console, usually opened with :kbd:`F12`,
+ to view **debug information** like JavaScript, engine, and WebGL errors.
+
+.. Attention::
+ Many browsers, Chromium-based browsers specifically, will not load exported
+ projects when **opened locally** per ``file://`` protocol. To get around this,
+ use a local server.
+
+ .. Tip::
```

Comment:
Lowercase + indentation of paragraph. (I usually indent all the way to be aligned with the end of the directive, but you can do 4-space indentation too).